dell/xps/13-9360/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, pkgs, ... }:
+let
+  firmware_qca6174 = pkgs.callPackage ./firmware_qca6174.nix {};
+in
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+  ];
+  boot.kernelModules = ["kvm-intel"]; # should this be in common/cpu/intel?
```

Comment:
The policy up until now was to not include options that are generated by `nixos-generate-config`.